docs/catalog.md

```diff
@@ -0,0 +1,117 @@
+| `stores`        | Array of Objects | Information about where the feedstock is stored. | `id`: ID of the store (Type: String)<br>`name`: Name of the store (Type: String, optional)<br>`url`: URL of the store (Type: String)<br>`rechunking`: Rechunking information (Type: Array of Objects, optional)<br>`public`: Whether the store is public (Type: Boolean, optional)<br>`geospatial`: Whether the store is geospatial (Type: Boolean, optional) |
+
+### Example YAML Files
+
+Here's an example of a `meta.yaml` file:
+
+```yaml
+# meta.yaml
+title: "LEAP Data Library Prototype"
+description: >
+  A prototype test for the LEAP Data Library refactor
+recipes:
+  - id: "small"
+    object: "recipe:small"
+  - id: "large"
+    object: "recipe:large"
+provenance:
+  providers:
+    - name: "Julius"
+      description: "Just a guy testing some recipes. Nothing to see here."
+      roles:
+        - producer
+        - licensor
+  license: "Just a Test"
+maintainers:
+  - name: "Julius Busecke"
+    orcid: "0000-0001-8571-865X"
+    github: jbusecke
+```
+
+Here's an example of a `catalog.yaml` file:
+
+```yaml
+# catalog.yaml
+meta_yaml_url: "https://github.com/leap-stc/proto_feedstock/blob/main/feedstock/meta.yaml"
+tags:
+  - my-custom-tag
+  - zarr
+stores:
+  - id: "small"
+    name: "The cool small Proto Dataset"
+    url: "gs://leap-scratch/data-library/feedstocks/proto_feedstock/small.zarr"
+    rechunking:
+      - path: "gs://some-bucket/small.zarr"
+        use_case: "multiscales"
+  - id: "large"
+    name: "The even cooler large Proto Dataset"
+    url: "gs://leap-scratch/data-library/feedstocks/proto_feedstock/large.zarr"
+```
+
+## CLI Functionality
+
+The `leap-data-management-utils` package provides CLI functionality to validate and generate feedstock catalogs. To use the CLI, you need to have the package installed. You can install the package using the following command:
+
+```bash
+python -m pip install leap-data-management-utils[catalog]
+```
+
+### Validation
+
+To validate the catalog files, you can use the `validate` command and provide the path to the `catalog.yaml` file as an argument:
+
+```bash
+leap-catalog validate --single path/to/catalog.yaml
+```
+
+Example:
+
+```bash
+leap-catalog validate --single https://github.com/leap-stc/proto_feedstock/blob/main/feedstock/catalog.yaml
+```
+
+Output:
+
+```plaintext
+✅ Valid feedstock: (1)
+  📂 https://github.com/leap-stc/proto_feedstock/blob/main/feedstock/catalog.yaml
+```
+
+### How to Add a New Dataset to the LEAP Web Catalog
+
+To add a new dataset to the LEAP web catalog, follow these steps:
+
+1. **Create YAML Files**: Create `meta.yaml` and `catalog.yaml` files for your dataset as shown in the examples above. These can reside in a GitHub repository or any other location accessible via a URL.
+2. **Add Dataset URL**: Add the URL of your dataset's `catalog.yaml` file to this [file](https://github.com/leap-stc/data-management/blob/main/catalog/input.yaml).
```

Comment:
Don't know if it's worth it, but might be nice to have a note that they can edit that file by clicking on the 
<img width="42" alt="image" src="https://github.com/leap-stc/leap-data-management-utils/assets/22455466/0fcd5cc1-80e2-493b-a01c-2050dce5aa9d"> icon.